iroha/src/main.rs

```diff
@@ -9,7 +9,7 @@ const TRUSTED_PEERS_PATH: &str = "trusted_peers.json";
 const GENESIS: &str = "genesis";
 
 #[tokio::main]
-#[allow(clippy::non_ascii_literal)]
+#[allow(clippy::non_ascii_literal, clippy::unwrap_used)]
```

Comment:
But it is not used... And should not be...

Reply:
It is used inside tokio main macro

Reply:
Why didn't it fail here previously?